File: firstdup.rb



dupS= "asdfqwerzxcvasdf"
dupS=dupS.split("")
puts dupS.class
#tried to convert string to array.

def firstdup(dupS)
	hash = Hash.new(0)
	dupS.each do |iter|
		hash[iter] +=1
		if hash[iter] == 2
			puts iter
			break

		end
	end
	puts hash
end


firstdup(dupS)

File: firstdup.py


def firstdup(dupS):
	dic = {}
	for iter in range(len(dupS)):
		dic[iter]=dupS.count(dupS[iter])
		if dic[iter] == 2:
			print dupS[iter]
			break
	print dic



if __name__ == "__main__":
	dupS="asdfqwerzxcvasdf"
	firstdup(dupS)



































